curations/nuget/nuget/-/System.CodeDom.yaml

```diff
@@ -0,0 +1,8 @@
+coordinates:
+  name: System.CodeDom
+  provider: nuget
+  type: nuget
+revisions:
+  4.5.0:
+    licensed:
+      declared: MIT
```

Comment:
https://www.nuget.org/packages/System.CodeDom links to https://github.com/dotnet/corefx/blob/master/LICENSE.TXT which is MIT